src/libraries/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -248,6 +248,9 @@ public abstract partial class SymmetricAlgorithm : System.IDisposable
         protected virtual void Dispose(bool disposing) { }
         public abstract void GenerateIV();
         public abstract void GenerateKey();
+        public int GetCiphertextLengthCbc(int plaintextLength, System.Security.Cryptography.PaddingMode paddingMode = System.Security.Cryptography.PaddingMode.PKCS7) { throw null; }
+        public int GetCiphertextLengthCfb(int plaintextLength, System.Security.Cryptography.PaddingMode paddingMode = System.Security.Cryptography.PaddingMode.None, int feedbackSizeInBits = 8) { throw null; }
```

Comment:
Note: I deviated a bit from the API proposal on the name of the feedback size parameters here. The proposal was `feedbackSizeBits`, but I (optimistically) went for <i>In</i>Bits. 

There is [some existing APIs][1] to use `In`:

```c#
public static System.Security.Cryptography.RSA Create (int keySizeInBits);
```

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsa.create?view=netcore-3.1#System_Security_Cryptography_RSA_Create_System_Int32_

I'm happy to change it back to the approved proposal, but thought the consistency would be preferable.

Reply:
![image](https://media1.tenor.com/images/ce411c531cef9905dc05aa51b1ca8112/tenor.gif?itemid=18137386)